packages/core/admin/admin/src/pages/Home/components/ContentManagerWidgets.tsx

```diff
@@ -48,14 +48,71 @@ const DocumentStatus = ({ status = 'draft' }: DocumentStatusProps) => {
   );
 };
 
+const WidgetContent = ({ document }: { document: RecentDocument }) => {
+  const { formatMessage } = useIntl();
+  const { trackUsage } = useTracking();
+  const navigate = useNavigate();
+
+  const handleRowClick = (document: RecentDocument) => () => {
+    trackUsage('willEditEntryFromHome');
+    const link = getEditViewLink(document);
+    navigate(link);
+  };
+
+  return (
+    <Tr onClick={handleRowClick(document)} key={document.documentId}>
+      <Td>
+        <CellTypography variant="omega" textColor="neutral800">
+          {document.title}
+        </CellTypography>
```

Comment:
Could you add a title attribute with document.title as the value? This way we'd get the browser's native tooltip to view the full content when it's too long to be displayed. In the other widget too, I realize I forgot it